### PR TITLE
Add missing metadata for bosun and scollector

### PR DESCRIPTION
**Pull request by gbrayut**

### CODE REVIEW

File: cmd/bosun/conf/notify.go [hunk 1]
```diff
@@ -12,8 +12,18 @@ import (
 
 	"bosun.org/_third_party/github.com/jordan-wright/email"
 	"bosun.org/collect"
+	"bosun.org/metadata"
 )
 
+func init() {
+	metadata.AddMetricMeta(
+		"bosun.email.sent", metadata.Counter, metadata.PerSecond,
+		"Counter of the number of times Bosun has sent an email notification.")
```
Comment (kylebrandt) [inline cmd/bosun/conf/notify.go]: "The number of email notifications sent by Bosun." , "The number of email notifications that Bosun failed to send."

File: cmd/bosun/sched/sched.go [hunk 2]
```diff
@@ -540,6 +540,12 @@ const (
 
 func init() {
 	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes, "The total size of the bosun state file.")
+	metadata.AddMetricMeta("bosun.check.duration", metadata.Gauge, metadata.Second, "The number of seconds it took Bosun to check each alert rule.")
```
Comment (kylebrandt) [inline cmd/bosun/sched/sched.go]: Why word wrap above and not here? (I like the wrapping personally).

File: cmd/bosun/sched/sched.go [hunk 3]
```diff
@@ -540,6 +540,12 @@ const (
...
+	metadata.AddMetricMeta("bosun.check.err", metadata.Gauge, metadata.Error, "Running count of the number of errors Bosun has received while trying to evaluate an alert expression.")
+	metadata.AddMetricMeta("bosun.ping.resolved", metadata.Gauge, metadata.None, "1=Ping resolved to an IP Address. 0=Ping failed to resolve to an IP Address.")
+	metadata.AddMetricMeta("bosun.ping.rtt", metadata.Gauge, metadata.MilliSecond, "Number of milliseconds for the echo reply to be received. Also know as Round Trip Time.")
```
Comment (kylebrandt) [inline cmd/bosun/sched/sched.go]: know , known. Also complete sentences for these, ie "The running count..". Metadata should not be None for resolve, should be Bool or Ok.

File: cmd/bosun/web/web.go [hunk 4]
```diff
@@ -41,6 +41,10 @@ const (
 func init() {
 	miniprofiler.Position = "bottomleft"
 	miniprofiler.StartHidden = true
+	metadata.AddMetricMeta("bosun.search.puts_relayed", metadata.Counter, metadata.Request, "Counter of api put requests sent to Bosun for relaying to the backend server.")
```
Comment (kylebrandt) [inline cmd/bosun/web/web.go]: The count of

File: cmd/bosun/sched/sched.go (hunk 2 above)
Comment (gbrayut) [inline cmd/bosun/sched/sched.go]: Existing line wasn't word wrapped. When we have multiple conventions I do my best to follow all of them :-P

File: cmd/bosun/sched/sched.go [hunk 5]
```diff
@@ -539,7 +539,20 @@ const (
 )
 
 func init() {
-	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes, "The total size of the bosun state file.")
+	metadata.AddMetricMeta("bosun.statefile.size", metadata.Gauge, metadata.Bytes,
+		"The total size of the bosun state file.")
```
Comment (mjibson) [inline cmd/bosun/sched/sched.go]: Capitalize `Bosun` to be consistent with other uses.

**Comment by gbrayut** [comment]
Changed bosun to Bosun, also created issue #906 for another inconsistent naming I found (there are probably more)

**Pull request merged**